js/ui/scroll_view/ui.scrollable.native.js

```diff
@@ -145,6 +145,7 @@ const NativeStrategy = Class.inherit({
 
     handleScroll: function(e) {
         // NOTE: ignoring scroll events when scroll location was not changed (for Android browser - B250122)
+        this._component._updateRtlConfig();
```

Comment:
imho "// NOTE: ignoring scroll events" should be inside the 'if(!this._isScrollLocationChanged()) {'

Reply:
Thank you for your attention.
Fixed https://github.com/DevExpress/DevExtreme/pull/14919/commits/9e7177e93126466fe94baa879b005386cbb60bbc